src/AutoRest.CSharp/Common/Output/Expressions/ValueExpressions/LiteralExpression.cs

```diff
@@ -1,7 +1,7 @@
-﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 namespace AutoRest.CSharp.Common.Output.Expressions.ValueExpressions
 {
-    internal record LiteralExpression(string Literal, bool U8) : ValueExpression;
+    internal record LiteralExpression(object Value) : ValueExpression;
```

Comment:
Why?

Reply:
here I am not changing this - I rename the previous `LiteralExpression` to `StringLiteralExpression` (you could see there is a new file called `StringLiteralExpression` which inherits this one) because I need a literal expression for a more generic type such as numbers or booleans, which are widely used in the example values.

Reply:
Remove this generic literal expression
We could use the FormattableStringToExpression as workaround for now